recipe/meta.yaml

```diff
@@ -28,7 +28,9 @@ requirements:
   run:
     - python
     - dataclasses >=0.6  # [py<37]
-    - typing-extensions >=3.7.4.3
+    # optional deps
```

Comment:
I would remove `- dataclasses >=0.6` as it does not apply anymore.

Reply:
To clarify, we are removing this because we no longer support anything less than Py 3.7, correct?

Reply:
Yes, and the upstream project no longer supports python 3.6 too.